python/dgl/dataloading/fixed.py

```diff
@@ -0,0 +1,146 @@
+"""Fixed subgraph sampler."""
+from ..sampling.utils import EidExcluder
+from .base import set_node_lazy_features, set_edge_lazy_features, Sampler
+
+# import non-DGL libraries
+import numpy as np
+import torch
+from collections import defaultdict
+
+class FixedSampler(Sampler):
```

Comment:
Not sure whether FixedSampler is a good name or not, maybe CappedNeighborSampler?

Reply:
Updated to `CappedNeighborSampler`, can also change the file name.